src/components/PostInfo/PostInfo.tsx

```diff
@@ -1,5 +1,49 @@
 import React from 'react';
+import { Post } from '../../types/Post';
+import { CommentList } from '../CommentList';
+import { UserInfo } from '../UserInfo';
 
-export const PostInfo: React.FC = () => (
-  <>Put the post here</>
+type Props = {
+  post: Post
+};
+
+export const PostInfo: React.FC<Props> = (
+  {
+    post: {
+      title, body, user, comments,
+    },
+  },
+) => (
```

Comment:
I would recommend you make destruction in the component instead of doing it in the arguments
Usually we don't make destruction in a component arguments